guide/src/io/csv_reader.md

```diff
@@ -16,28 +16,18 @@ This crate relies on [the crate `csv`](https://crates.io/crates/csv) to scan and
 As an example, the following infers the schema and reads a CSV by re-using the same reader:
 
 ```rust
-use arrow2::io::csv::read;
-use arrow2::error::Result;
-
-fn read_path(path: &str) -> Result<()> {
-    let mut reader = read::ReaderBuilder::new().from_path(path)?;
-    let parser = read::DefaultParser::default();
-
-    let schema = read::infer_schema(&mut reader, None, true, &infer)?;
-
-    // 0: start from
-    // 100: up to (max batch size)
-    let batch = read::read_batch(&mut reader, &parser, 0, 100, schema, None)?;
-}
+{{#include ../../../examples/csv_read.rs}}
```

Comment:
didn't know we can embed examples in the docs now. Very cool

Reply:
Yeah, makes it very easy because people can run the example via cargo, but also understand its context on the guide.